contracts/cw721-stake/src/contract.rs

```diff
@@ -525,3 +526,10 @@ pub fn query_staked_nfts(
 
     to_binary(&res)
 }
+
+#[cfg_attr(not(feature = "library"), entry_point)]
+pub fn migrate(deps: DepsMut, _env: Env, _msg: MigrateMsg) -> Result<Response, ContractError> {
+    // Set contract to version to latest
+    set_contract_version(deps.storage, CONTRACT_NAME, CONTRACT_VERSION)?;
+    Ok(Response::default())
+}
```

Comment:
How would you feel about adding some really simple tests for these? Could probably just write one like (pseudocode):

```
let mut deps = mock_dependencies();
cw2::set_contract_version(deps.storage, "mock", "mock").unwrap();
migrate(deps.as_mut(), mock_env(), MigrateMsg {}).unwrap();
let version = cw2::get_contract_version(deps.storage).unwrap();
assert_eq!(version.version, env!("CARGO_PKG_VERSION");
```

and copy it into all of the contracts. would catch if we forget about this again.

Reply:
yess good call